apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1562,6 +1562,61 @@
                         }
                     }
                 },
+                "currency" : {
+                    "type" : "object",
+                    "additionalProperties" : false,
+                    "properties" : {
+                        "long_name": {
```

Comment:
perhaps `long_name` should be just `name` and `short_name` should be symbol 🤔  just a suggestion 💭 

I'm not sure if the properties schema has been agreed somewhere else already or this is first proposition